hazelcast/src/test/java/com/hazelcast/partition/impl/MigrationQueueTest.java

```diff
@@ -0,0 +1,77 @@
+package com.hazelcast.partition.impl;
+
+import com.hazelcast.test.annotation.QuickTest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+@Category(QuickTest.class)
+public class MigrationQueueTest {
+
+    private final MigrationQueue migrationQueue = new MigrationQueue();
+
+    @Test
+    public void test_migrationTaskCount_incremented_onMigrateTask() {
```

Comment:
Why does every test have 'test_migrationTaskCount' as prefix? It doesn't seem to add a lot of additional info,